src/OrchardCore.Modules/OrchardCore.Media/OrchardCore.Media.csproj

```diff
@@ -14,17 +14,17 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\..\OrchardCore\OrchardCore.Admin.Abstractions\OrchardCore.Admin.Abstractions.csproj" />
```

Comment:
Order alphabetically (I am such a PITA)

Reply:
Done. No worries, with a bit more effort I might just break the 100 commit barrier!